src/isar/apis/models/start_mission_definition.py

```diff
@@ -80,7 +81,10 @@ def to_isar_mission(
     if start_mission_definition.start_pose:
         start_pose = start_mission_definition.start_pose.to_alitra_pose()
 
+    id = start_mission_definition.id if start_mission_definition.id else uuid4_string()
+
     return Mission(
+        id=id,
```

Comment:
I agree to this change, and the line 84 here can be removed, as the default value will be set to a uuid4_string in the constructor of the StartMissionDefinition class.

We might want to do input validation on that id to verify that it is an uuid coming in.

```suggestion
    return Mission(
        id=start_mission_definition.id,
```

Reply:
If we want validation we should make a separate issue for it as that would make sense to implement other places as well, like task id